src/core/server/core_usage_data/core_usage_data_service.test.ts

```diff
@@ -274,6 +292,301 @@ describe('CoreUsageDataService', () => {
         `);
       });
     });
+
+    describe('getConfigsUsageData', () => {
```

Comment:
Can we also test the scenario where the key is an array of objects? An example is Maps' `regionmap.layers` config.